### PR TITLE
feat: RND23 Add Mr Men copy to footer on CR.com

### PR DESCRIPTION
**Pull request by michael-odonovan**
#### What is it doing?
Creates a reusable component for corporate sponsor logo or similar on every page, using our footer.

#### Why is this required?
RN23

#### link to Jira ticket:
https://comicrelief.atlassian.net/browse/ENG-2046

### Quick Checklist:
- [x] My PR title follows the Conventional Commit spec.

- [x] I have filled out the PR description as per the template above.

- [x] I have added tests to cover new or changed behaviour.

- [x] I have updated any relevant documentation.

### CODE REVIEW

File: src/components/Organisms/Footer/Footer.js [hunk 1]
```diff
@@ -15,14 +15,16 @@ import {
...
+  navItems, footerCopy, campaign, additionalLegalLine, ...rest
 }) => {
   // Remove white space between words
   const campaignName = campaign.replace(/\s/g, '').toLowerCase();
+
   return (
     <div>
       <FooterWrapper navItems {...rest}>
         <InnerWrapper>
+          <Text style={{ textAlign: 'left' }} align="left" tag="p" color="white">{additionalLegalLine}</Text>
```
Comment (pauldolden) [inline src/components/Organisms/Footer/Footer.js:27]: Might be worth making this conditional, as currently the tag is showing appearing in the DOM but with no content:

```jsx
{additionalLegalLine && <Text style={{ textAlign: 'left' }} align="left" tag="p" color="white">{additionalLegalLine}</Text>}
          
```

```html
<p color="white" class="sc-gsTEea jZslOC"></p>
```

Comment (pauldolden) [inline src/components/Organisms/Footer/Footer.js:27]: Could add align to `Text` component.

**Review by pauldolden — CHANGES_REQUESTED** [review]
Nice work, just a couple of little things I've noticed.

File: src/components/Organisms/Footer/Footer.js (hunk 1 above)
Comment (pauldolden) [reply src/components/Organisms/Footer/Footer.js]: Infact ignore this.

**Review by AndyEPhipps — APPROVED** [review]

**Review by pauldolden — APPROVED** [review]
🚀

**Pull request merged**